### PR TITLE
Update Individual Response button on Interstitial page

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
The `Request separate census` button on the Individual Response interstitial page should be a '[Call to action](https://ons-design-system.netlify.app/components/button/#call-to-action)' button with chevron, using the `button` type anchor link with URL.

### How to review 
Use the `test_individual_response` schema to check the change

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Change call to action on /individual-response/ launcher from form sumbit button to anchor link](https://trello.com/c/DwnsWQdo/4188-change-call-to-action-on-individual-response-launcher-from-form-sumbit-button-to-anchor-link)

File: templates/individual_response/interstitial.html [hunk 1]
```diff
@@ -17,6 +17,7 @@ <h1 class="u-mt-m">{{_("If you can't answer questions for others in your househo
         onsButton({
             "text": _("Request separate census"),
             "classes": 'btn u-mb-m u-mt-s btn--link',
+            "url": "",
```
Comment (ajmaddaford) [inline templates/individual_response/interstitial.html:20]: The chevron is missing from the button the as the `url` property is empty. For this to work we need to remove the `POST` from the endpoint and on a GET request determine the URL for the next page. That URL should be used here, and will be whatever we currently redirect to on the POST request.

File: app/views/handlers/individual_response.py [hunk 2]
```diff
@@ -150,6 +150,24 @@ def handle_get(self):
             self.individual_section_id
         )
 
+        self._list_name = self._schema.get_individual_response_list()
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:153]: `self._list_name` is already set in the init.
```suggestion
```

File: app/views/handlers/individual_response.py [hunk 3]
```diff
@@ -150,6 +150,24 @@ def handle_get(self):
...
+            )
+        elif len(list_model.non_primary_people) == 1:
+            next_location_url = url_for(
+                ".individual_response_how",
+                list_item_id=list_model.non_primary_people[0],
+                journey="hub",
+            )
+        else:
+            next_location_url = url_for(".individual_response_who", journey="hub")
+
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:170]: There is a lot going on in this method now. I think we should create `get_previous_location_url` and `get_next_location_url` methods, and call them from here.

File: tests/integration/individual_response/test_individual_response.py [hunk 4]
```diff
@@ -21,6 +21,10 @@ def individual_response_link(self):
         if response_paragraph:
             return response_paragraph.find_next()["href"]
 
+    def get_next_location_url_from_interstitial_page(self):
+        submit_button = self.getHtmlSoup().find("a", {"data-qa": "btn-submit"})
+        return submit_button.attrs["href"]
```
Comment (ajmaddaford) [inline tests/integration/individual_response/test_individual_response.py:26]: This method should be consistent with the `individual_section_link` and `individual_response_link` methods. It should be defined as a `@property` and named something like `individual_response_start_link`.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: app/views/handlers/individual_response.py [hunk 5]
```diff
@@ -160,59 +160,56 @@ def _publish_fulfilment_request(self, mobile_number=None):
...
-        if len(list_model.non_primary_people) == 1:
-            return redirect(
-                url_for(
-                    ".individual_response_how",
-                    list_item_id=list_model.non_primary_people[0],
-                    journey="hub",
-                )
-            )
-        return redirect(url_for(".individual_response_who", journey="hub"))
+            return url_for("questionnaire.get_questionnaire")
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:212]: Same as the comment above about the `elif` and `else`.

File: app/views/handlers/individual_response.py [hunk 6]
```diff
@@ -160,59 +160,56 @@ def _publish_fulfilment_request(self, mobile_number=None):
...
+                list_item_id=list_model.non_primary_people[0],
+                journey="hub",
+            )
+        else:
+            return url_for(".individual_response_who", journey="hub")
+
+    def _get_previous_location_url(self):
         individual_section_first_block_id = self._schema.get_first_block_id_for_section(
             self.individual_section_id
         )
```
Comment (ajmaddaford) [inline app/views/handlers/individual_response.py:194]: As this is only used in the second conditional, these lines should be moved there.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Review by LJBabbage — APPROVED** [review]
The moving around of stuff confusion me a touch, but it looks ok and works as expected

**Review by petechd — APPROVED** [review]

**Pull request merged**